packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -81,8 +145,8 @@ const tessenAction =
   };
 
 const createTessen = (config) => ({
-  page: tessenAction('page', config),
-  track: tessenAction('track', config),
+  page: tessenPage('page', config),
+  track: tessenTrack('track', config),
```

Comment:
do we need the `page` and `track` arguments now that we have specific functions?